hubmap/frontend/src/middleware/PubSub.js

```diff
@@ -0,0 +1,17 @@
+import PubSub from 'pubsub-js';
+
+export default class PubSubApi {
+tokens = {};
+
+subscribe = (topic, subscriber) => {
+  this.tokens[subscriber] = PubSub.subscribe(topic, subscriber);
+}
+
+publish = (topic, object) => {
+  PubSub.publish(topic, object);
+}
+
+unsubscribe = (token) => {
+  PubSub.unsubscribe(token);
+}
+}
```

Comment:
We might want to add redux+pubsub at some point. but I removed it for now, since i don't intend to implement it now. I would like to complete working on this PR for now. So I removed PubSub.js now.